### DISCUSSION TITLE
ıbax

### DISCUSSION THREAD

**Discussion by aysegulozbek** [IBAX-io/go-ibax#35] [Typos （English）]
![Ekran Alıntısı 1](https://user-images.githubusercontent.com/97479351/148840042-b02aea11-36f4-4f8c-9e01-a77abd154d78.PNG)
![Ekran Alıntısı2](https://user-images.githubusercontent.com/97479351/148840347-3a851252-1de1-4f04-8f1b-03343812653b.PNG)
![Ekran Alıntısı 3](https://user-images.githubusercontent.com/97479351/148840652-1d331c83-c3ba-4c74-92c0-1d6c5c77a759.PNG)
 
All available in pictures..

**Comment by HalloKyle** [comment]
Thank you for your support we will fix that shortly after our tech guy is free.
